docs/UserGuide.md

```diff
@@ -1020,18 +1020,18 @@ Some commands require you to include parameters. These parameters are identified
+| p\|    | Phone Number              | 30          | It can start with an optional `+` to indicate the country code, with the rest only containing numbers. It should be at least 3 digits long.                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| e\|    | Email                     | 100         | **Format:** local-part@domain<br/> **Constraints for local part:**<br/> • Should only contain alphanumeric characters, and the characters `+`, `_`, `.` and `-`.<br/> • Should not start or end with special characters.<br/> • Should not have two consecutive special characters.<br/> **Constraints for domain:**<br/> • Made up of domain labels separated by periods.<br/> • Must end with a domain label of at least 2 characters long.<br/> • Domain label should consist of alphanumeric characters separated only by singular hyphens, if any. |
+| a\|    | Address                   | 100         | Can take any values.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| i\|    | Path of CSV File to Import| -           | Should be the absolute file path of the CSV file without any inverted commas.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
Thanks for adding this, makes the constraints much more specific

Reply:
👍